lib/catalog/rbac/access.rb

```diff
@@ -1,8 +1,6 @@
 module Catalog
   module RBAC
     class Access
-      include Insights::API::Common::RBAC::Utilities
```

Comment:
Has this been 'extra weight' for a while? The removed code below doesn't appeart to depend on this either.

Reply:
@syncrou I removed it here from the common gem
RedHatInsights/insights-api-common-rails#166

This was after Erik's PR to move the validate group into a class by itself in
https://github.com/RedHatInsights/insights-api-common-rails/pull/164

We should have technically removed it here
https://github.com/RedHatInsights/catalog-api/pull/604